src/gui/qgsmapcanvas.cpp

```diff
@@ -889,7 +889,7 @@ void QgsMapCanvas::showContextMenu( QgsMapMouseEvent *event )
     }
   }
   copyCoordinateMenu->addSeparator();
-  QAction *setCustomCrsAction = new QAction( QStringLiteral( "Set Custom CRS…" ), mMenu );
+  QAction *setCustomCrsAction = new QAction( QString( tr( "Set Custom CRS" ) ).append( QChar( 0x2026 ) ), mMenu );
```

Comment:
```suggestion
  QAction *setCustomCrsAction = new QAction( tr( "Set Custom CRS…" ) ), mMenu );
```

That should be sufficient (and make sure it can be properly handled in every translation; e.g. not sure about RTL languages).